### DISCUSSION TITLE
What do the images on the scoring mean?

### DISCUSSION THREAD

**Discussion by LUKATHELONEWOLF** [janstehno/cotw-companion#51] [Questions] [answered]
Is there any way to make a key for the scoring what the images mean for that particular score on the app

**Comment by janstehno** [comment, marked as answer]
I can explain what I think is pretty much obvious.

Trophy score has 5 tiers. Bronze, silver, gold, diamond, and great one. Silver, gold, and diamond can be found in the animal's detailed info. And all of them in the logbook, whenever you are creating a new logbook entry. The icons for these are the same as the ones in the game. Great one is slightly different. The icons are also colored based on the rating. Brown for bronze, grey for silver, yellow for gold, blue for diamond, and white for great one.

As for the maximum trophy score in the animal's detailed info, left side shows **maximum trophy score**. On the right side, there is a **maximum weight**. If the animal can be a great one, there is also a second line underneath. On the left side, there is a **maximum trophy score** when the animal is a great one. And, you can guess, on the right side, **maximum weight** when the animal is a great one.

If you meant anything else, please let me know. I hope you can now understand the information and icons a little more.
